doc/rst/source/gmtconnect.rst

```diff
@@ -58,10 +58,12 @@ Optional Arguments
 .. _-C:
 
 **-C**\ [*closed*]
-    Write all the closed polygons to *closed* [gmt connect_closed.txt]
-    and all other segments as they are to stdout. No connection takes
-    place. Use **-T**\ *cutoff* to set a minimum separation [0], and if
-    *cutoff* is > 0 then we also explicitly close the polygons on output.
+    Write all the already-closed polygons to file *closed* [gmt connect_closed.txt]
```

Comment:
Is `gmt connect_closed.txt` correct?

Reply:
No, I fixed many of these but that one slipped through.  Fixed.